gpc-ping/DTO/V074RequestingPractitioner.cs

```diff
@@ -0,0 +1,23 @@
+namespace gpc_ping;
+
+public class V074RequestingPractitioner : RequestingPractitioner
+{
+    public string ResourceType { get; set; }
+    public PractitionerRole[] PractitionerRole { get; set; } = [];
+}
+
+public class PractitionerRole
+{
+    public Role? Role { get; set; }
```

Comment:
Are we certain that the role of the practitioner is nullable? I suppose it could be (for system-level accounts); however, I don't believe this is good practice - it is preferable for a user (and their role) to be indicated where possible. Might be worth checking with Steve Hunt to discuss scenarios where practitioner role would be null (if any), if you haven't already.